proposals/POEM-N-template.md

```diff
@@ -0,0 +1,49 @@
+This is a template for a filed proposal.
+Create a pull request using this template.
+
+# Title
+This is the title of the POEM.
+A good simple title would help to describe the purpose of a proposal.
+
+## Status
+* Current state: (Draft | In-progress | Completed | Abandoned)
+* Author: @author (Github ID)
+
+## Summary
+
+This section summarize the proposal.
+A brief description, proposed changes, and effects are expected to be included.  
+ 
+
+## Motivation
+
+Please describe the problem to be solved by this proposal and/or any ideas behind it.
+
+## Proposed changes
+This section may include large subsections, diagrams, links to references, and so on.
+
+### Architecture Diagram (optional) 
+
+It is recommended to include an architectural diagram.
+A link to an external resource is enough.
+
+
+### Design
+
+This section describes the design details.
+
+
+### Implementation details
+
+This section describe how to implement the proposal. 
+
+
+## Issue (optional)
+
+Any issue(compatibility, drawbacks, etc) should be describe here.
+
+## Integration and Migration plan (optional)
```

Comment:
Really like the fact that this is included in the template, usually this is one of the most important parts as software is being used on productions systems by a vendor with a large based of end users